src/executor/evaluate/mod.rs

```diff
@@ -286,5 +286,27 @@ async fn evaluate_function<'a, T: 'static + Debug>(
             Nullable::Null => Ok(Value::Null),
         }
         .map(Evaluated::from),
+        Function::Ltrim { expr, chars } | Function::Rtrim { expr, chars } => {
+            let name = if matches!(func, Function::Ltrim { .. }) {
+                "LTRIM"
+            } else {
+                "RTRIM"
+            };
+            let pattern: Result<Vec<char>> = match eval_to_str(name, chars).await? {
+                Nullable::Value(v) => Ok(v.chars().collect::<Vec<char>>()),
+                Nullable::Null => Ok(" ".chars().collect::<Vec<char>>()),
```

Comment:
I just realized that if the 2nd arg `pattern` is Null, the result should be Null 😞

Reply:
And empty string should do nothing..

Reply:
Oh, I also didn't catch that, good point!
Then, AST would be.. like this?
```rust
pub enum Function {                           
    ...
    Ltrim { expr: Expr, chars: Option<Expr> },
    Rtrim { expr: Expr, chars: Option<Expr> },
}
```

Reply:
I guess so. But It's weird current code works with the test cases without Option<>

Reply:
That sounds like good time to add more test cases :)